board-report-2022-07.md

```diff
@@ -0,0 +1,46 @@
+## Description:
+
+The mission of Apache Arrow is the creation and maintenance of
+software related to columnar in-memory processing and data interchange
+
+## Issues:
+
+There are no issues requiring board attention.
+
+## Membership Data:
+
+Apache Arrow was founded 2016-01-20 (6 years ago)
+There are currently 76 committers and 39 PMC members in this project.
+The Committer-to-PMC ratio is roughly 5:3.
+
+Community changes, past quarter:
+- No new PMC members. Last addition was QP Hou on 2022-02-17.
+- Alenka Frim was added as committer on 2022-06-23
+- Dewey Dunnington was added as committer on 2022-06-15
+- Rok Mihevc was added as committer on 2022-06-15
+- L. C. Hsieh was added as committer on 2022-04-28
+
+## Project Activity:
+
+Recent releases:
+
+- RS-17.0.0 was released on 2022-06-28.
+- RS-16.0.0 was released on 2022-06-13.
+- RS-15.0.0 was released on 2022-05-30.
+- RS-14.0.0 was released on 2022-05-16.
+- 8.0.0 was released on 2022-05-07.
+- RS-13.0.0 was released on 2022-05-03.
+- RS-12.0.0 was released on 2022-04-19.
+- RS-DF-7.1.0 was released on 2022-04-18.
+- JULIA-2.3.0 was released on 2022-04-12.
+
+## Community Health:
+
+- A book about Apache Arrow was published from a community member.
+- We've split Apache Arrow Ballista to
+  https://github.com/apache/arrow-ballista from
+  https://github.com/apache/arrow-datafusion .
+- We've marked Apache Arrow C Streaming Interface stable.
+- A conference that focuses on Apache Arrow was held by community members.
```

Comment:
Can we add the link to the conference (or the youtube channel)? Just a suggestion, it shall make it easy for the reader to explore further if interested. Same suggestion for book name